packages/logs/src/domain/assembly.ts

```diff
@@ -26,12 +26,9 @@ export function startLogsAssembly(
     ({ rawLogsEvent, messageContext = undefined, savedCommonContext = undefined, domainContext }) => {
       const startTime = getRelativeTime(rawLogsEvent.date)
       const session = sessionManager.findTrackedSession(startTime)
+      const shouldSendLog = sessionManager.findTrackedSession(startTime, { returnInactive: true })
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏So much easier now!